app/javascript/components/base/base-search-component.vue

```diff
@@ -0,0 +1,41 @@
+<template>
+  <div class="flex my-4 items-center justify-start">
+    <form method="GET">
+      <div class="relative text-yellow-700">
+        <span class="absolute inset-y-0 left-0 flex items-center pl-2">
+          <button
+            type="submit"
+            class="p-1 focus:outline-none focus:shadow-outline"
+          >
+            <svg
+              fill="none"
+              stroke="currentColor"
+              stroke-linecap="round"
+              stroke-linejoin="round"
+              stroke-width="2"
+              viewBox="0 0 24 24"
+              class="w-6 h-6 text-yellow-700 "
+            >
+              <path d="M21 21l-6-6m2-5a7 7 0 11-14 0 7 7 0 0114 0z" />
+            </svg>
+          </button>
+        </span>
+        <input
+          type="search"
+          name="q"
+          class="border-2 w-full border-yellow-700 py-2 text-sm text-yellow-700 pl-10 focus:outline-none focus:bg-white focus:text-yellow-700"
+          placeholder="Buscar ingrediente..."
+          autocomplete="off"
+        >
+      </div>
+    </form>
+  </div>
+</template>
+
+<script>
+export default {
+  props: {
+    ingredient: { type: Object, required: true },
+  },
+};
+</script>
```

Comment:
🚫 **[eslint]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Newline required at end of file but not found  eol-last